j2cl-maven-plugin/src/main/java/com/vertispan/j2cl/mojo/TestMojo.java

```diff
@@ -244,6 +244,9 @@
     @Parameter(defaultValue = "false")
     protected boolean enableSourcemaps;
 
+    @Parameter
```

Comment:
docs? can follow up in a later release, but it should be the same as BuildMojo

Reply:
technically this property should be added to WatchMojo too, so that it is doc'd correctly, and can be set at the parent - see how languageOut, checkAssertions, rewritePolyfills are all declared but unused

Reply:
are we going to run WatchMojo in advanced mode ? anyway, i ll add it

Reply:
Yes, that's a thing, I'm not going to tell people they _can't_ do it...

Reply:
Reading it now, it looks like you did miss WatchMojo?